custom-recipes/googlesheets-append/recipe.py

```diff
@@ -52,7 +58,9 @@ def append_rows(self, values, value_input_option='RAW'):
 
     return self.spreadsheet.values_append(self.title, params, body)
 
-ws.append_rows = append_rows.__get__(ws, ws.__class__)
+
+worksheet.append_rows = append_rows.__get__(worksheet, worksheet.__class__)
+
```

Comment:
It would be nice to bump the version of gspread at some point so we can get rid of this patch (not sure of the implications of doing that though)